### ISSUE TITLE
[iPad] Add horizontal side margins to tableviews for iPad

### ISSUE THREAD

**Issue by jwe-apple** [ResearchKit/ResearchKit#77]
In the iPad introductory screens, table views have horizontal margins of 115pt on each side. For ResearchKit, we would like to apply the same margins for table views and in fact probably most other vertically scrolling content as well.

Example:

![img_0003](https://cloud.githubusercontent.com/assets/11575859/7264445/4391cb0a-e840-11e4-9e37-8c90d05ae702.PNG)

**Comment by rsanchezsaez** [comment]
Should the same be done for iPhone 6+?

**Comment by jwe-apple** [comment]
No, not for the 6+.

**Comment by rsanchezsaez** [comment]
Should maybe all the different *step views* have the margins? Which view do you think should hold these margins? Maybe they should be applied to `ORKVerticalContainerView`.

**Comment by jwe-apple** [comment]
At a first pass, I think they should apply to ORKVerticalContainerView, and also to cases where we're using ORKTableContainerView.

A slight complication to the ORKVerticalContainerView implementation is that the stepView for the vertical container should probably be laid out full width, but with autolayout margins. That way if a stepView needs to be wider, for example for a cognitive game, it can. I believe there is already some code in ORKVerticalContainerView which does something similar, so hopefully the patch won't be too messy. The autolayout code in there is a bit tangled.

**Comment by rsanchezsaez** [comment]
Implemented in [pull request #108](https://github.com/ResearchKit/ResearchKit/pull/108).

I tried to maintain the current system for calculating view layouts. Layout code is currently a little bit over the place. Maybe it would be a good idea to refactor some of the layout code from some cells/views to the more general superclasses. If you agree, maybe let's create a new issue for the layout code refactor (would be a good chance for standardizing constraints handling as well).
